fcc-test-suite-template-master/index.html

```diff
@@ -30,7 +30,7 @@ <h1><center>Here's a time line of Dr. César's life:</center></h1>
         <ul>
```

Comment:
### Great comments ✨ however, it only lacks a little bit of explanation and it can be improved 🚀 

- [ ] As reviewers we also focus on things the student did well on his code so it will be great if you highlight some of the work that the student did well 💪🏼 👏🏼 
- [ ] Kindly describe to the student the `why` he should make the requested changes so the changes make sense and are fair 👍🏼 💯 
- [ ] Examples are stronger than words in conveying messages🎯, so you can add an example To ensure that the student understands what changes he needs to make like 🥇 
- [ ] This is **[Optional]** but highly recommended in order for your comments to sound more human-friendly please add some emojis to your comments 🤖 👽